### PR TITLE
Update UG.md

### PR DESCRIPTION
**Pull request by Aishwarya-Hariharan-Iyer**
Update the UG to contain commands planned for v1.1 of DevEnable.

Based on our team document linked [here](https://docs.google.com/document/d/1_iJyfbqPF-lV3i2GC0nB_dKrSA7zzxvQ7D_Smc7o-Oc/edit?usp=sharing).

#### Things to do:

- Picture of UI to be added
- Commands to be reviewed after implementation

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -45,125 +45,186 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
 
 Examples:
-* `list` followed by `delete 2` deletes the 2nd person in the address book.
-* `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
+*  `client -t 1 n/Amy` Tags the project with `PROJECT_ID` 1 with a client with `CLIENT_NAME` Amy.
+*  `client -t 2 n/Bob p/91234567` Tags the project with `PROJECT_ID` 2 with a client with `CLIENT_NAME` Bob and 
+   `CLIENT_CONTACT_NUMBER` 91234567.
+*  `client -t 3 n/Charlie e/charlie@gmail.com` Tags the project with `PROJECT_ID` 2 with a client with `CLIENT_NAME` 
+    Charlie and `CLIENT_EMAIL` charlie@gmail.com.
+*  `client -t 2 n/Dave p/91111111 e/dave@gmail.com`
```
Comment (crvstalphua) [inline docs/UserGuide.md:141]: i think the explanation was missed out also 
```suggestion
*  `client -t 2 n/Dave p/91111111 e/dave@gmail.com` Tags the project with `PROJECT_ID` 2 with a client with `CLIENT_NAME` Dave, `CLIENT_CONTACT_NUMBER` 91111111 and `CLIENT_EMAIL` dave@gmail.com
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -45,125 +45,186 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
-* The index **must be a positive integer** 1, 2, 3, …​
+* Adds the client to the project with the specified `PROJECT_ID`.
 
 Examples:
-* `list` followed by `delete 2` deletes the 2nd person in the address book.
-* `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
+*  `client -t 1 n/Amy` Tags the project with `PROJECT_ID` 1 with a client with `CLIENT_NAME` Amy.
+*  `client -t 2 n/Bob p/91234567` Tags the project with `PROJECT_ID` 2 with a client with `CLIENT_NAME` Bob and 
+   `CLIENT_CONTACT_NUMBER` 91234567.
+*  `client -t 3 n/Charlie e/charlie@gmail.com` Tags the project with `PROJECT_ID` 2 with a client with `CLIENT_NAME`
```
Comment (crvstalphua) [inline docs/UserGuide.md:139]: i think the project ID in the explanation is supposed to be 3
```suggestion
*  `client -t 3 n/Charlie e/charlie@gmail.com` Tags the project with `PROJECT_ID` 3 with a client with `CLIENT_NAME` 
```

**Review by crvstalphua — CHANGES_REQUESTED** [review]
Noticed just two minor details that we missed out previously but other than that LGTM.

File: docs/UserGuide.md (hunk 1 above)
Comment (Aishwarya-Hariharan-Iyer) [reply docs/UserGuide.md]: Made changes. Thank you!

**Review by ConradLew — APPROVED** [review]
LGTM.

**Review by crvstalphua — APPROVED** [review]
LGTM

**Pull request merged**